src/txdb.h

```diff
@@ -70,6 +81,14 @@ class CBlockTreeDB : public CDBWrapper
     bool ReadReindexing(bool &fReindex);
     bool ReadTxIndex(const uint256 &txid, CDiskTxPos &pos);
     bool WriteTxIndex(const std::vector<std::pair<uint256, CDiskTxPos> > &list);
+    bool UpdateAddressUnspentIndex(const std::vector<CAddressUnspent> &vect);
+    bool ReadAddressUnspentIndex(uint160 addressHash, int type,
+                                 std::vector<CAddressUnspent> &vect);
+    bool WriteAddressIndex(const std::vector<CAddressAmount> &vect);
+    bool EraseAddressIndex(const std::vector<CAddressAmount> &vect);
+    bool ReadAddressIndex(uint160 addressHash, int type,
+                          std::vector<CAddressAmount> &addressIndex,
+                          int start = 0, int end = 0);
```

Comment:
Oops, not a big problem, but I just noticed I can use my new typedefs in some of the function prototypes above, to simplify them, will do that in a later force-push.